Classes/Eel/Helper/TrackingConfiguration.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Neos\GoogleAnalytics\Eel\Helper;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\ContentRepository\Domain\NodeAggregate\NodeName;
+use Neos\ContentRepository\Domain\Projection\Content\NodeInterface;
+use Neos\Eel\ProtectedContextAwareInterface;
+use Neos\Utility\Arrays;
+
+/**
+ * Class TrackingConfiguration
+ *
+ * @package Neos\GoogleAnalytics\Eel\Helper
+ */
+class TrackingConfiguration implements ProtectedContextAwareInterface
+{
+    /**
+     * @Flow\InjectConfiguration(path="default", package="Neos.GoogleAnalytics")
+     * @var array
+     */
+    protected $defaultSettings;
+
+    /**
+     * @Flow\InjectConfiguration(path="sites", package="Neos.GoogleAnalytics")
+     * @var array
+     */
+    protected $sitesSettings;
+
+
+    /**
+     * @param $site
+     * @param string $path
+     * @return array
+     */
+    public function setting($site, string $path)
```

Comment:
I would prefer having this logic in a Fusion prototype so it could be modified easily (f.e. reading config from a homepage property) and would apply in all prototypes using it. What do you think?

Reply:
Unfortunately, there is no EEL helper for merging arrays recursively (yet). So, I think we should keep it in this helper for now.

I have refactored the Fusion code a bit, so the configuration can be overridden in the Base prototype.